content/en/docs/contributing/release-process.md

```diff
@@ -27,14 +27,16 @@ The process for cutting a minor release is as follows:
 
 1. Ensure upgrading document exists.
 
-2. Ensure all strings of versions have been updated (TODO `@joshvanl`: update this
-   in new documentation tree).
+2. Ensure all strings of versions have been updated.
 
 - `deploy/charts/cert-manager/README.md`
```

Comment:
Add a note after: `(in the [jetstack/cert-manager](https://github.com/jetstack/cert-manager) repository)`